docs/references.md

```diff
@@ -0,0 +1,31 @@
+---
+title: References & Bibliography
+sidebar_label: References
+---
+
+
+The following normative documents contain provisions, which, through reference in this text, constitute provisions of this Standard. For dated references, subsequent amendments to, or revisions of, any of these publications do not apply. However, parties to agreements based on this Standard are encouraged to investigate the possibility of applying the most recent editions of the normative documents indicated below. For undated references, the latest edition of the normative document referred to applies:
+
+- **Apache 2.0 open-source license**, [https://www.apache.org/licenses/LICENSE-2.0].
+- **Community Specification license**, [https://github.com/CommunitySpecification/1.0]
+- **JSON Schema**, [https://json-schema.org/].
+- **OpenAPI Standard v3.0**, [https://www.openapis.org/].
+- **RFC 2119**, _Keywords for use in RFCs to Indicate Requirement Levels, March 1997_, [https://datatracker.ietf.org/doc/html/rfc2119].
+- **RFC2782**, _A DNS RR for specifying the location of services (DNS SRV), February 2000_, [https://datatracker.ietf.org/doc/html/rfc2782].
```

Comment:
Should be consistent here - either with or without a space between RFC and RFC number.

Reply:
good point, fixed